Source/ActivityPub.Types/AS/Collection/ASCollection.cs

```diff
@@ -112,8 +112,14 @@ public int TotalItems
     /// <summary>
     ///     True if this is a paged collection, false otherwise.
     /// </summary>
+    [MemberNotNullWhen(true, nameof(FirstPopulated))]
     public bool IsPaged => Current != null || First != null || Last != null;
 
+    /// <summary>
+    ///     In a paged Collection, gets the first populated entity.
+    /// </summary>
+    public Linkable<ASCollectionPage>? FirstPopulated => IsPaged ? Current ?? First ?? Last : null;
```

Comment:
We should be able to remove the IsPaged check and simplify it down to just `Current ?? First ?? Last`